build.sbt

```diff
@@ -42,3 +42,14 @@ lazy val lambda = (project in file("lambda"))
     riffRaffManifestProjectName := "MemSub::Subscriptions::Lambda::PriceMigrationEngine",
     riffRaffArtifactResources += (project.base / "cfn.yaml", "cfn/cfn.yaml")
   )
+
+lazy val stateMachine = (project in file("stateMachine"))
+  .enablePlugins(RiffRaffArtifact)
+  .settings(
+    name := "price-migration-engine-state-machine",
+    description := "Cloudformation for price migration state machine.",
+    riffRaffPackageType := (baseDirectory.value / "cfn"),
```

Comment:
I think you will need to move the cfn.yaml into a cfn/ subdirectory to get riff-raff to pick this up. 

It took a bit of fiddling about to get this to work in the dynamoDB project

Reply:
That's already done.  It's the move: `stateMachine/cfn.yaml → stateMachine/cfn/cfn.yaml`

Reply:
Ah sorry :)